android/app/src/main/kotlin/app/revanced/manager/flutter/ExportSettingsActivity.kt

```diff
@@ -0,0 +1,87 @@
+package app.revanced.manager.flutter
+
+import android.app.Activity
+import android.content.Context
+import android.content.Intent
+import android.content.pm.PackageInfo
+import android.content.pm.PackageManager
+import android.os.Bundle
+import android.util.Base64
+import org.json.JSONObject
+import java.io.ByteArrayInputStream
+import java.io.File
+import java.security.cert.CertificateFactory
+import java.security.cert.X509Certificate
+import java.security.MessageDigest
+
+class ExportSettingsActivity : Activity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        val callingPackageName = getCallingPackage()!!
+
+        if (getFingerprint(callingPackageName) == getFingerprint(getPackageName())) {
+            // Create JSON Object
+            val json = JSONObject()
+
+            // Default Data
+            json.put("keystorePassword", "s3cur3p@ssw0rd")
```

Comment:
What if I changed the password?

Reply:
@BenjaminHalko I believe there's an variable in settings for the keystore password, it can be used here

Reply:
Then it will get changed in when it loops through all the settings in shared preferences. This is in case the user doesn't change the password and an entry for the password is not in shared preferences. All other settings use the Compose default if the entry is not in Flutters shared preferences but since the password is required for importing the keystore, I put the default password in just in case Compose every decides to change its default password.

Reply:
@Ushie wouldn't that require loading Flutter? This activity doesn't use flutter as it is unnecessary.